src/server/csp.ts

```diff
@@ -65,7 +65,7 @@ const buildCsp = ({ styles, scripts }: Assets, twitter: boolean): string => `
     script-src 'self' ${assetHashes(scripts)} http://www.instagram.com/embed.js https://interactive.guim.co.uk https://s16.tiktokcdn.com https://www.tiktok.com/embed.js ${twitter ? 'https://platform.twitter.com https://cdn.syndication.twimg.com' : ''};
     frame-src https://www.theguardian.com https://www.scribd.com https://www.instagram.com https://www.tiktok.com https://interactive.guim.co.uk https://open.spotify.com https://www.youtube-nocookie.com ${twitter ? 'https://platform.twitter.com https://syndication.twitter.com https://twitter.com' : ''};
     font-src 'self' https://interactive.guim.co.uk;
-    connect-src 'self' https://interactive.guim.co.uk https://sf-hs-sg.ibytedtos.com/ https://gdn-cdn.s3.amazonaws.com/
+    connect-src 'self' https://callouts.code.dev-guardianapis.com/formstack-campaign/submit https://interactive.guim.co.uk https://sf-hs-sg.ibytedtos.com/ https://gdn-cdn.s3.amazonaws.com/
```

Comment:
Hmm, this is an interesting point. Perhaps we need to start passing the `Stage` to the CSP module to swap these URLs between CODE and PROD. An example of switching CSPs like this can be seen in the support-frontend confs: [CODE.public.conf](https://github.com/guardian/support-frontend/blob/f7ba7175bd75368a33ee61296ef87903670841a4/support-frontend/conf/CODE.public.conf#L20) and [PROD.public.conf](https://github.com/guardian/support-frontend/blob/f7ba7175bd75368a33ee61296ef87903670841a4/support-frontend/conf/PROD.public.conf#L19).

Reply:
That could be a good idea, in this case (confusingly) both PROD and CODE submit to that url. It's the same case on dotcom @JustinPinner

Reply:
That's... odd. Why would PROD submit to a CODE url?